Clarify tutorial creates a new msg type (#294)
micro-ROS/micro-ROS.github.io: _docs/tutorials/core/create_new_type/index.md

---

> title: How to create a new micro-ROS message type

Reviewer: ```suggestion
title: How to create custom message types for micro-ROS
```

Reviewer: @ralph-lange if you are ok, commit this suggestion and I'll merge

> title: How to create a new micro-ROS message type

Author: ```suggestion
title: Adding support for custom ROS message types to micro-ROS
```

Author: I believe this would be more descriptive and accurate than the other two suggestions.

Reviewer 2: This title is a bit long in my opinion. In principle, the support is always there. It's really about creating a custom message type in an own project.

> title: How to create a new micro-ROS message type

Reviewer 3: ```suggestion
title: How to add support for custom ROS message types to micro-ROS
```

Reviewer: ok

Author: This seems similar to my suggestion, which @ralph-lange wasn't happy with.

Perhaps "Including a custom ROS message in micro-ROS"?

That doesn't imply special support is needed (although I believe with the way ROS 2 and micro-ROS work, it is actually almost like adding support), and does seem to communicate that *those* are the steps you'd need to get your own custom messages (or someone else's) included in your local micro-ROS build.

Reviewer 2: I like you proposal, @gavanderhoorn! It is short, precise and already implies in the title that the procedure is similar to that for custom message types in normal ROS 2.

Author: Ok, so here it is as a suggestion:

```suggestion
title: Including a custom ROS message in micro-ROS
```

Author: Only thing is, it doesn't mention services nor actions, so I could see some users getting confused when reading this, but not sure we can help that other than making it clear in the tutorial text itself this is the procedure to follow for those as well.

Reviewer: I'm ok with @gavanderhoorn suggestion. I assume that @ralph-lange also is. Please @FranFin commit and merge if you are ok.
